UITests/ViewMemoryLeakTests.swift

```diff
@@ -26,14 +26,18 @@ class ViewMemoryLeakTests: KIFTestCase, UITextFieldDelegate {
 
     func testAboutHomeDisposed() {
         // about:home is already active on startup; grab a reference to it.
-        let browserViewController = UIApplication.sharedApplication().keyWindow!.rootViewController!
+        let browserViewController = getTopViewController()
         weak var aboutHomeController = getChildViewController(browserViewController, childClass: "HomePanelViewController")
 
-        // Change the page to make about:home go away.
-        tester().tapViewWithAccessibilityIdentifier("url")
-        let url = "\(webRoot)/numberedPage.html?page=1"
-        tester().clearTextFromAndThenEnterTextIntoCurrentFirstResponder("\(url)\n")
-        tester().waitForWebViewElementWithAccessibilityLabel("Page 1")
+        // We have to use an autorelease pool here or else the weak reference to the about:home controller won't get cleared
+        // right away although the strong reference on the browser controller is set to nil.
+        autoreleasepool {
```

Comment:
This is weird and I don't really have an explanation for why only using an autorelease pool makes the weak reference turn `nil`. There must be an autoreleased reference flying around somewhere but I could not see it. Another (less probable) explanation is that this is a bug in ARC. The [official docs](https://developer.apple.com/library/prerelease/ios/documentation/Swift/Conceptual/Swift_Programming_Language/AutomaticReferenceCounting.html) state

> [...] ARC automatically sets a weak reference to nil when the instance that it refers to is deallocated. [...] you will never end up with a reference to an invalid instance that no longer exists.